From 673fcc044e4e6099a3855e75b8cef34a9667a87c Mon Sep 17 00:00:00 2001
From: James Jackson <superjax08@gmail.com>
Date: Wed, 22 Apr 2020 08:58:03 -0700
Subject: [PATCH] fix build

---
 include/async_comm/comm.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/async_comm/comm.h b/include/async_comm/comm.h
index f24286c..1391744 100644
--- a/include/async_comm/comm.h
+++ b/include/async_comm/comm.h
@@ -163,7 +163,7 @@ private:
     uint8_t data[READ_BUFFER_SIZE];
     size_t len;
 
-    ReadBuffer(const uint8_t * buf, size_t len) : len(len)
+    ReadBuffer(const uint8_t * buf, size_t _len) : len(_len)
     {
       assert(len <= READ_BUFFER_SIZE); // only checks in debug mode
       memcpy(data, buf, len);
@@ -178,7 +178,7 @@ private:
 
     WriteBuffer() : len(0), pos(0) {}
 
-    WriteBuffer(const uint8_t * buf, size_t len) : len(len), pos(0)
+    WriteBuffer(const uint8_t * buf, size_t _len) : len(_len), pos(0)
     {
       assert(len <= WRITE_BUFFER_SIZE); // only checks in debug mode
       memcpy(data, buf, len);
-- 
2.17.1

